noria/Cargo.toml

```diff
@@ -44,6 +44,7 @@ tracing-futures = "0.2.2"
 slab = "0.4"
 pin-project = "0.4.0"
 futures-util = "0.3.0"
+mysql = "18.1.0"
```

Comment:
I _think_ this can use `mysql_common` instead, which is much smaller, and also has [`Value`](https://docs.rs/mysql_common/0.21.0/mysql_common/value/enum.Value.html).

Reply:
You are correct, clicking on the **src** of `mysql::Value` redirected to... `mysql_common::value::Value`. I got duped :P